src/ambianic/pipeline/store.py

```diff
@@ -174,16 +177,35 @@ def notify(self, save_json: dict):
         if self.notification is None:
             return
         log.debug("Sending notification(s)..")
+        directory = "{0}/ambianic-edge/premium.yaml".format(os.getcwd())
+
         # TODO extract inference data
         for inference_result in save_json['inference_result']:
-            data = {
-                'id': save_json['id'],
-                'label': inference_result['label'],
-                'confidence': inference_result['confidence'],
-                'datetime': save_json['datetime'],
-            }
-            notification = Notification(data=data, providers=self.notification_config["providers"])
-            self.notification.send(notification)
+            try: 
```

Comment:
@vickywane why hardcode the assumption that the edge device needs premium subscription. What if the user does not want to have premium subscription at all. This code should only trigger if the user has elected to sign up for premium services and consequently the PWA has informed the edge devices about that. Otherwise this code should not be part of the edge flow at all.

Reply:
In fact I think we should move to a completely independent local library all the logic related to turning on premium subscriptions. It will be likely used for functions other than notifications in the future.

Reply:
Okay. 

Can this come in a follow-up PR when we have more use cases for the premium subscription?

Reply:
For now, the `store.py` has been readjusted to accommodate non-premium users.

As illustrated in the design diagram, obtaining the `ACCESS_TOKEN` is the last step in the premium notification flow, hence, the premium notification flow is only used when there is an `ACCESS_TOKEN` field in the `premium.yaml` file.

Reply:
> @vickywane why hardcode the assumption that the edge device needs premium subscription. What if the user does not want to have premium subscription at all. This code should only trigger if the user has elected to sign up for premium services and consequently the PWA has informed the edge devices about that. Otherwise this code should not be part of the edge flow at all.

The premium notification flow has been re-adjusted to fix this concern. Rather than re-implementing a new notification flow with `aprise`, the premium notification flow would add the needed credentials to the `config.yaml` file so it can be used by the existing `NotificationHandler`, to avoid duplicating code.